Package.swift

```diff
@@ -22,7 +22,7 @@
 
 import PackageDescription
 
-let firebaseVersion = "6.34.0"
+let firebaseVersion = "7.0.0"
```

Comment:
I can't comment on it since it's existing code but `Package.swift` also needs to remove the `FIRCore_VERSION` and the rest.